js/src/components/adaptive-form/adaptive-form.js

```diff
@@ -27,7 +27,7 @@ function isEvent( value ) {
  * several workarounds in order to be compatible with WC 6.9 to 7.1.
  *
  * @param {Object} props React props.
- * @param {JSX.Element | (formProps: Object) => JSX.Element} props.children Children to be rendered. Could be a render prop function.
+ * @param {JSX.Element | Function} props.children Children to be rendered. Could be a render prop function.
```

Comment:
Swapping the order could make it parsable as well.

```suggestion
 * @param {(formProps: Object) => JSX.Element | JSX.Element} props.children Children to be rendered. Could be a render prop function.
```